RevenueCatUI/Templates/V2/Components/Icon/IconComponentView.swift

```diff
@@ -56,9 +56,8 @@ struct IconComponentView: View {
                    shadow: style.iconBackgroundShadow,
                    background: style.iconBackgroundStyle,
                    uiConfigProvider: self.viewModel.uiConfigProvider)
-            .padding(style.padding)
+            .padding(style.margin)
             .size(style.size)
-            .clipped()
```

Comment:
I can't see a reason why the icon needs to be clipped. This is likely to clip the shadow if present, which is not intended.

Reply:
Copy/paste from image 🫠